### DISCUSSION TITLE
Turbine position misaligned

### DISCUSSION THREAD

**Discussion by jacktang** [NatLabRockies/floris#674] [Q&A] [answered]
Hello,

I learnt `06_sweep_wind_conditions` tutorial, and tried to all wake visualization of the farm. But got below result:
![image](https://github.com/NREL/floris/assets/44341/99321633-fa3a-4235-8580-92b9899f4208)

The turbines are misaligned. And the code I added:

```
fig, ax = plt.subplots(1, 1, figsize=(5,3), dpi=400)
horizontal_plane = fi.calculate_horizontal_plane(wd=[wd_array[0]], ws=[ws_array[0]], height=90.0)
wakeviz.visualize_cut_plane(
    horizontal_plane,
    ax=ax,
    title="Farm",
    wd=[wd_array[0]],
    
)
wakeviz.plot_turbines_with_fi(fi, ax, wd=[wd_array[0]], 
                               color="black"
                              )
```
Can you tell me how to fix this? Thanks!

**Comment by Bartdoekemeijer** [comment, marked as answer]
Hi, this has been solved in #663. You can checkout the `develop` branch of FLORIS to pull in the fix immediately, or wait until a new version is released with this fix.
